game-app/game-core/src/main/java/games/strategy/engine/data/PlayerList.java

```diff
@@ -20,9 +22,19 @@ public class PlayerList extends GameDataComponent implements Iterable<GamePlayer
 
   // maps String playerName -> PlayerId
   private final Map<String, GamePlayer> players = new LinkedHashMap<>();
+  @Getter private final GamePlayer nullPlayer;
 
   public PlayerList(final GameData data) {
     super(data);
+    nullPlayer =
+        new GamePlayer(Constants.PLAYER_NAME_NEUTRAL, true, false, null, false, data) {
+          private static final long serialVersionUID = 1;
+
+          @Override
+          public boolean isNull() {
+            return true;
+          }
+        };
```

Comment:
Did you consider making this a simple "getter"method without a field that backs it, instead creating a new instance on every call?